packages/terra-dynamic-grid/src/styles.js

```diff
@@ -0,0 +1,110 @@
+// IE < 11 doesn't honor window.CSS.Supports. Edge has supports, but will fail on grid.
+const cssGridsEnabled = window.CSS && window.CSS.supports && window.CSS.supports('display', 'grid');
+
+const gridTemplateColumns = (layout) => {
+  const columns = layout['grid-template-columns'];
+  const msColumns = columns.split(/\s+/).join(` ${layout['grid-gap'] || 0} `);
```

Comment:
So I dont think we should support `grid-column-gap` and `grid-row-gap` but rather support shorthand syntax of `grid-gap: <grid-column-gap> <grid-row-gap>` This is easier to implement and doesnt' take away any functionality.